Environment/solaredge.5m.py

```diff
@@ -25,11 +25,21 @@
 # Optional. Set either as empty string to disable.
 awake_icon = "☀︎"
 asleep_icon = "☾"
+battery_icon = "🔋"
+low_battery_icon = "🪫"
+
+# Optional. Set to 0 to disable, 1 to enable. Display current battery charge state
+battery_present = 1
```

Comment:
This is a good candidate for [xbar variables](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html). There are [Python examples](https://github.com/search?q=repo%3Amatryer%2Fxbar-plugins+xbar.var+language%3APython&type=code&l=Python) in the repo to follow.